src/main/java/sg/ncl/MainController.java

```diff
@@ -1822,6 +1844,36 @@ public Experiment2 extractExperiment(String experimentJson) {
         return experiment2;
     }
 
+    private Realization invokeAndExtractRealization(Long id) {
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("Authorization", AUTHORIZATION_HEADER);
+
+        HttpEntity<String> request = new HttpEntity<String>("parameters", headers);
+        ResponseEntity respEntity = restTemplate.exchange(properties.getRealization(id.toString()), HttpMethod.GET, request, String.class);
+
+        return extractRealization(respEntity.getBody().toString());
+    }
+
+    private Realization extractRealization(String json) {
+        Realization realization = new Realization();
+        JSONObject object = new JSONObject(json);
+
+        realization.setExperimentId(object.getLong("experimentId"));
+        realization.setExperimentName(object.getString("experimentName"));
+        realization.setUserId(object.getString("userId"));
+        realization.setTeamId(object.getString("teamId"));
+        realization.setState(object.getString("state"));
+
+        String exp_report = object.get("details").toString();
+//        String[] exp_report_parts = exp_report.split("@");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)